packages/documentation/src/content/docs/integration/deployment/services/backend-service.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Backend service
+---
+
+import { LinkOut } from '@interledger/docs-design-system'
+
+Rafiki’s <LinkOut href="https://github.com/interledger/rafiki/tree/main/packages/backend">`backend`</LinkOut> service is the software’s main service for handling business logic and external communication. The service is responsible for:
+
+- Exposing the endpoints of the [Open Payments APIs](#open-payments) for public clients to perform account management tasks. This is the external communication piece.
+- Exposing an [Interledger connector](#interledger-connector) to send and receive STREAM packets with peers.
+- Enabling you to manage accounts and track balances.
+- Exposing an internal [GraphQL Backend Admin API](#graphql-backend-admin-api) for service operators to manage accounts and application settings, like peering relationships.
+
+## Requirements
+
+The following are required when using the `backend` service.
+
+- A Redis database as a cache to share STREAM connection details across processes
+- A Postgres database, separate from the `auth` service’s database, for Open Payments resources
+- TigerBeetle or a separate Postgres database for accounting balances at the ILP layer
+  :::note
+  You can use the same database instance of Postgres for the `backend` service, `auth` service, and accounting balances (if not using TigerBeetle). However, separate database schemas are required within the Rafiki instance to maintain boundaries between the objects being managed.
+  :::
+
+You must also set the environment variables for the `backend` service.
```

Comment:
Should we link to those?

Reply:
Leaving this comment open as a reminder. We plan to include the environment variables for each service within the service's page, then have one big resource page that contains all the variables.